src/utils/Constants.js

```diff
@@ -22,5 +22,6 @@ export const USER_NAME_KEY = "userName";
 export const AUTH_KEY = "authKey";
 export const PASSWORD_KEY = "password";
 export const IP_ADDRESS_KEY = "ipAddress";
+export const LOGIN_TOKEN = "login_token";
```

Comment:
Can you rename this as this is the standard format we are doing it?
`export const LOGIN_TOKEN = "loginToken";` (Camel case instead of underscore

Reply:
The problem with renaming the string is that it must be changed from the server side too as the "login_token" is passed to the browser. A workaround may be to use "login_token" as strings instead of using a constant but that defeats the whole purpose of creating a constant variable, probably.